packit.yml

```diff
@@ -0,0 +1,29 @@
+---
+# https://packit.dev/docs/configuration/
+
+specfile_path: simple-term.spec
+files_to_sync: [simple-term.spec, packit.yml]
+upstream_package_name: simple-term
+downstream_package_name: simple-term
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets: [fedora-all]
+
+  - job: tests
+    trigger: pull_request
+    targets: [fedora-all]
+
+  - job: copr_build
+    trigger: commit
+    branch: main
+    targets: [fedora-all]
+    owner: mh21
+    project: simple-term-latest
+
+  - job: copr_build
+    trigger: release
+    targets: [fedora-all]
+    owner: mh21
+    project: simple-term
```

Comment:
I think you/we should make up our minds -- build on commits *or* on pushing a tag. Having both doesn't make sense IMHO.

Reply:
one is for the releases - a.k.a. stable, the other is latest main

Reply:
Oh right, you have *two* COPRs, I missed that. Sorry!